pkg/csi/recover/recover.go

```diff
@@ -151,42 +154,61 @@ func (r *FuseRecover) runOnce() {
 	r.recover()
 }
 
-func (r FuseRecover) recover() {
+func (r *FuseRecover) recover() {
 	brokenMounts, err := mountinfo.GetBrokenMountPoints()
 	if err != nil {
 		glog.Error(err)
 		return
 	}
 
 	for _, point := range brokenMounts {
-		glog.V(4).Infof("Get broken mount point: %v", point)
+		if lock := r.locks.TryAcquire(point.MountPath); !lock {
```

Comment:
How about using r.checkAndRecoverMounts() to wrap the logic in the loop? We can use defer `r.locks.Release` to make the logic simper.

Reply:
Thx for the suggestion. Moved it into `func doRecover`